testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_no_conflicts_cbl.py

```diff
@@ -577,6 +577,18 @@ def test_sg_CBL_updates_concurrently(params_from_base_test_setup, sg_conf_name,
     c = cluster.Cluster(config=cluster_config)
     c.reset(sg_config_path=sg_config)
 
+    if sync_gateway_version >= "2.5.0":
+        sg_client = MobileRestClient()
+        expvars = sg_client.get_expvars(sg_admin_url)
+        num_replications_active = expvars["syncgateway"]["per_db"][sg_db]["database"]["num_replications_active"]
+        num_replications_total = expvars["syncgateway"]["per_db"][sg_db]["database"]["num_replications_total"]
+        """if continuous:
```

Comment:
waiting for the bugs to be fixed. ONce bugs are fixed, need to verify whether having continuous and without continuous replication matters